apps/console/src/features/applications/components/modals/application-share-modal.tsx

```diff
@@ -67,6 +73,9 @@ export const ApplicationShareModal: FunctionComponent<ApplicationShareModalProps
                     level: AlertLevels.SUCCESS,
                     message: "Application shared!"
                 }));
+                eventPublisher.publish("application-share", {
```

Comment:
Improve formatting. Check the other places also.

eventPublisher.publish(
          "application-share",
          { "client-id": clientId }
);

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/3829/commits/6117b186f38e79e6357bf2e324673ab7fe4525e3

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/3829/commits/6117b186f38e79e6357bf2e324673ab7fe4525e3